lib/connection/terminal.coffee

```diff
@@ -12,16 +12,18 @@ module.exports =
         console.log err
 
   term: (sh) ->
-    if process.platform == "darwin"
-      @exec "osascript -e 'tell application \"Terminal\" to activate'"
-      @exec "osascript -e 'tell application \"Terminal\" to do script \"#{@escape(sh)}\"'"
-    else if process.platform == "windows"
-      @exec "cmd /C start cmd /C #{@escape(sh)}"
-    else if process.platform == "linux"
-      # probably debian-specific
-      @exec "x-terminal-emulator -e \"#{@escape(sh)}\""
-    else
-      console.log 'unsupported platform'
+    switch process.platform
+      when "darwin"
+        @exec "osascript -e 'tell application \"Terminal\" to activate'"
+        @exec "osascript -e 'tell application \"Terminal\" to do script \"#{@escape(sh)}\"'"
+      when "windows"
+        @exec "cmd /C start #{@terminal()} /C #{@escape(sh)}"
```

Comment:
I was thinking of putting the whole `cmd /C start cmd /C` as part of the config. Probably also makes sense to include the flags, just in case alternatives need different ones.